governance.md

```diff
@@ -0,0 +1,175 @@
+# The United States Research Software Engineer (US-RSE) Association
+
+## Membership and Governance
+
+Date: June XX, 2020
```

Comment:
How about we just pick today? 
```suggestion
Date: June 12, 2020
```

Reply:
Perhaps the date we get everyone to approve?

Reply:
I suggest we use the date this is merged into the repo. Whoever does the merge should just make one final commit to update this date.

Reply:
This date would be updated every time there is a merge.

Reply:
OK. I like the suggestion that whoever merges updates the date.